CONTRIBUTING.md

```diff
@@ -73,12 +73,13 @@ and additionally the following rules:
   an explanation why the behaviour should differ. You can try to negate the logic
   check. As alternative consider the approach documented in
   [ui-framework-documentation.md](ui-framework-documentation.md)
-* Consider a corresponding Tumbleweed test: Because of
-  [Factory First](https://opensource.suse.com/suse-open-source-policy)
-  we should test against Tumbleweed by default (with exceptions for special
-  packages, of course). So please check if your code changes work with a current
-  Tumbleweed snapshot and then add to the according schedule for Tumbleweed
-  tests, e.g. main.pm
+* Support openSUSE Tumbleweed as primary product: Because of
```

Comment:
Still `should` and `please`, which makes this the weakest item in the list. What about just:

```Because of Factory First, Tumbleweed is the default test target.```

Reply:
yes, good. Rephrased and left out the politeness ;) Please check again

Reply:
Perfect!